webpackjs/components/DocklessData/MonthSelect.js

```diff
@@ -15,21 +15,15 @@ const months = [
   "December",
 ];
 
-// To do: Pass getDaysInMonth function in to this component from index.js
-// as a prop instead of defining it in both places.
-const getDaysInMonth = (month, year) => {
-  return new Date(year, month, 0).getDate();
-}
 // Get the range of months starting when we first have today to this current
 // month as an array of objects. Ex: [{1_2019: 'January 2019'}, ...]
-const getMonthOptions = () => {
+const getMonthOptions = ( getDaysInMonth ) => {
```

Comment:
Passing in this `getDaysInMonth` function as an argument to another function is a little overly complex. Could you just move this `getMonthOptions` function inside the scope of the `MonthSelect` functional component?

So...

```js
const MonthSelect = ({ monthYear, getDaysInMonth, onChangeMonth }) => {
   function getMonthOptions () { // no argument passing needed since we're inside the scope
    ...
    let lastDay = getDaysInMonth(`${index + 1}`, `${year}`);
   ...
  };

  return (
    ... // all the JSX thats already inside the component
  );
}
```

Reply:
YES! This is what I wanted to do in the first place, but I didn't know the best way to go about it. Consider it done - this will also come in very handy in the future.